cmd/kubeadm/app/cmd/init.go

```diff
@@ -112,12 +113,30 @@ func NewCmdInit(out io.Writer) *cobra.Command {
 		"etcd client key file. Note: The path must be in /etc/ssl/certs",
 	)
 	cmd.PersistentFlags().MarkDeprecated("external-etcd-keyfile", "this flag will be removed when componentconfig exists")
+	cmd.PersistentFlags().BoolVar(
+		&skipChecks, "skip-checks", false,
```

Comment:
`skip-checks` has a clearer intent than `skip-preflight`. As an user, if I didn't know already what you're talking about, I'd ask _what's preflight?_.
I'm ok with `skip-preflight-checks`, though.